prepare.yml

```diff
@@ -186,7 +186,7 @@
       tags: ndt-e2e
       environment:
         http_proxy: "{{ http_proxy }}"
-      raw: "git clone {{ ndt_e2e_client_git }} {{ ndt_e2e_client_dir }}"
+      raw: "git clone --branch master {{ ndt_e2e_client_git }} {{ ndt_e2e_client_dir }}"
```

Comment:
Are there times when HEAD points to a branch other than master? This seems like a fairly rare circumstance, at least as far as I've seen us develop at M-Lab.